docs/discover/get-started-discover.asciidoc

```diff
@@ -0,0 +1,356 @@
+
+. Click **Add filter**.
+. Continue your exploration by adding more filters.
+. To remove a filter, click the close icon (x) next to its name in the filter bar.
+
+[float]
+==== Search and filter using ES|QL
+
+You can use **Discover** with the Elasticsearch Query Language, ES|QL. When using ES|QL,
+you don't have to select a data view. It's your query that determines the data to explore and display in Discover.
+
+You can switch to the ES|QL mode of Discover from the application menu bar.
+
+Note that in ES|QL mode, the **Documents** tab is named **Results**.
+
+Learn more about how to use ES|QL queries in <<try-esql, Using ES|QL>>.
+
+
+
+[float]
+[[save-discover-search]]
+==== Save your search for later use
+
+Save your search so you can use it later, generate a CSV report, or use it to create visualizations, dashboards, and Canvas workpads.
+Saving a search saves the query text, filters,
+and current view of *Discover*, including the columns selected in
+the document table, the sort order, and the {data-source}.
+
+. In the application menu bar, click **Save**.
+
+. Give your search a title and a description.
+
+. Optionally store <<managing-tags,tags>> and the time range with the search.
+
+. Click **Save**.
+
+[float]
+[[share-your-findings]]
+==== Share your search
+
+To share your search and **Discover** view with a larger audience, click *Share* in the application menu bar.
+For detailed information about the sharing options, refer to <<reporting-getting-started,Reporting>>.
+
+
+[float]
+[[alert-from-Discover]]
+=== Generate alerts
+
+From *Discover*, you can create a rule to periodically
+check when data goes above or below a certain threshold within a given time interval.
+
+. Ensure that your data view,
+query, and filters fetch the data for which you want an alert.
+. In the application menu bar, click *Alerts > Create search threshold rule*.
++
+The *Create rule* form is pre-filled with the latest query sent to {es}.
+. <<rule-type-es-query, Configure your query>> and <<action-types, select a connector type>>.
+
+. Click *Save*.
+
+For more about this and other rules provided in {alert-features}, go to <<alerting-getting-started>>.
+
+
+[float]
+=== What’s next?
+
+* <<discover-search-for-relevance, Search for relevance>>.
+
+* <<document-explorer, Configure the chart and document table>> to better meet your needs.
+
+[float]
+=== Troubleshooting
+
+This section references common questions and issues encountered when using Discover.
+Also check the following blog post: {blog-ref}troubleshooting-guide-common-issues-kibana-discover-load[Learn how to resolve common issues with Discover.]
+
+**Some fields show as empty while they should not be, why is that?**
+
+This unexpected situation can happen with runtime fields and regular keyword fields when the string exceeds the value set for the {ref}/ignore-above.html[ignore_above] setting used when indexing the data into {es}.
```

Comment:
```suggestion
This could be the case with runtime fields and regular keyword fields when the string exceeds the value set for the {ref}/ignore-above.html[ignore_above] setting used when indexing the data into {es}.
```

It's expected from ES perspective. Not a bug.

Reply:
Should we also mention another case we are aware of? If users have `nested` fields, values would be available only for the root field and empty for the "leaf" fields https://github.com/elastic/kibana/issues/190768

Reply:
If it helps users, yes we should :D Can we do this separately from this PR? Or would you be able to draft a quick explanation?

Reply:
```suggestion
This can happen with runtime fields and regular keyword fields when the string exceeds the value set for the {ref}/ignore-above.html[ignore_above] setting used when indexing the data into {es}.
```
Just re-rewording the suggestion you made above. Thanks! (unexpected was from the user's perspective, but I guess we should not assume what they're thinking 😅)

Reply:
Something like: "To see values of nested fields, add the field with `nested` type to the table instead of its sub fields."